group_trigger.go

```diff
@@ -200,20 +190,21 @@ func (broker *TriggerBroker) Distribute(primaries map[int]triggerList) (map[int]
 	for idx, tlist := range primaries {
 		broker.latestPrimaries[idx] = tlist.frames
 		nprimaries += len(tlist.frames)
-		err := broker.triggerCounters[idx].observeTriggerList(&tlist)
+		err := broker.triggerCounters[idx].countNewTriggers(&tlist)
 		if err != nil {
 			log.Printf("triggering assumptions broken!\n%v\n%v\n%v", err,
```

Comment:
I was going to suggest changing this panic to just log, which it seems you did.... so good

Reply:
Don't think this was ever a panic, at least not recently.